README.md

```diff
@@ -10,7 +10,7 @@ All supported versions are listed below. For every supported version, acceptance
 
 Supported operating systems are listed in `metadata.json` but individual releases can divert from that. For example, if Pulpcore x.y drops EL7, it will still be listed in metadata.json until all versions supported by the module have dropped it. Similarly, if x.z adds support for EL9, it'll be listed in `metadata.json` and all versions that don't support EL9 will have a note.
 
-### Pulpcore 3.13
+### Pulpcore 3.14
```

Comment:
This should be a separate patch IMHO. For example, 3.14 also means we should update our CI.

Reply:
Works for me, I wanted to capture that this change requires this update. Should @wbclark do tha tin the worker PR or should it be the first PR, stand-alone, that goes in?

Reply:
For what it's worth, the alternative is to mention that it does nothing on 3.13 (where unknown options are just ignored).